Dockerfile

```diff
@@ -1,17 +0,0 @@
-FROM python:3.8-slim-buster
```

Comment:
We should remove other redundant files/folders such as `k8s/` with the Helm charts etc.

Reply:
Done